app/models/Classroom.coffee

```diff
@@ -32,3 +32,59 @@ module.exports = class Classroom extends CocoModel
     }
     _.extend options, opts
     @fetch(options)
+    
+  getLevels: (options={}) ->
+    # options: courseID, withoutLadderLevels
+    Levels = require 'collections/Levels'
```

Comment:
I like this; we should do this more for files where not everything needs the thing being required